subsys/infinispan/src/main/java/org/commonjava/indy/subsys/infinispan/config/ISPNRemoteConfiguration.java

```diff
@@ -43,6 +43,8 @@
 
     private String remotePatterns;
 
+    private String hotrodClientConfigPath;
```

Comment:
With this new approach, are the old remoteServer, hotrodPort, remotePatterns fields still needed?

Reply:
These fields are not needed any more, I will remove them to avoid confusion.